components/org.wso2.carbon.identity.context.rewrite.valve/src/main/java/org/wso2/carbon/identity/context/rewrite/valve/TenantContextRewriteValve.java

```diff
@@ -81,6 +81,10 @@ public void invoke(Request request, Response response) throws IOException, Servl
             }
         }
 
+        if (isTenantQualifiedUrlsEnabled()) {
```

Comment:
why do we need to check this configuration?

Reply:
I think we don't need to check this config.

Reply:
added the check since we get the tenant domain from the ThreadLocal only when this config is enabled.